bug: daClient.SetKeyringKeyname undefined (type *"github.com/celestiaorg/celestia-openrpc".Client has no field or method SetKeyringKeyname) undefined: regexp

# Summary

After adding keyname, caused some issues.

```log
../das/celestia.go:158:17: daClient.SetKeyringKeyname undefined (type *"github.com/celestiaorg/celestia-openrpc".Client has no field or method SetKeyringKeyname)
../das/celestia.go:723:29: undefined: regexp
```

## Full logs

```log
# on main before last PR merged
js@jss-MacBook-Air nitro-das-celestia % cd cmd && go build -o celestia-server
js@jss-MacBook-Air cmd % git pull
remote: Enumerating objects: 1, done.
remote: Counting objects: 100% (1/1), done.
remote: Total 1 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
Unpacking objects: 100% (1/1), 914 bytes | 457.00 KiB/s, done.
From github.com:jcstein/nitro-das-celestia
   348ccb5..749e265  main       -> origin/main
Updating 348ccb5..749e265
Fast-forward
 Dockerfile      |  2 +-
 README.md       |  9 ++++++++-
 das/celestia.go | 18 ++++++++++++++++++
 3 files changed, 27 insertions(+), 2 deletions(-)

# after PR merged
js@jss-MacBook-Air cmd % go build -o celestia-server 
# github.com/celestiaorg/nitro-das-celestia/das
../das/celestia.go:158:17: daClient.SetKeyringKeyname undefined (type *"github.com/celestiaorg/celestia-openrpc".Client has no field or method SetKeyringKeyname)
../das/celestia.go:723:29: undefined: regexp
```

## Idea to consider

Clean up to keyName and KeyName

--- a/Dockerfile
+++ b/Dockerfile
@@ -64,4 +64,4 @@ USER ${USER_NAME}
 
 # Expose ports:
 EXPOSE 1317 9090 26657 1095 8080 26658
-ENTRYPOINT ["sh", "-c", "/bin/celestia-server --enable-rpc --rpc-addr $RPC_ADDR --rpc-port $RPC_PORT --celestia.auth-token $AUTH_TOKEN --celestia.gas-price $GAS_PRICE --celestia.gas-multiplier $GAS_MULTIPLIER --celestia.namespace-id $NAMESPACEID --celestia.rpc $CELESTIA_NODE_ENDPOINT --celestia.keyring-keyname $KEYNAME"]
+ENTRYPOINT ["sh", "-c", "/bin/celestia-server --enable-rpc --rpc-addr $RPC_ADDR --rpc-port $RPC_PORT --celestia.auth-token $AUTH_TOKEN --celestia.gas-price $GAS_PRICE --celestia.gas-multiplier $GAS_MULTIPLIER --celestia.namespace-id $NAMESPACEID --celestia.rpc $CELESTIA_NODE_ENDPOINT --celestia.keyname $KEYNAME"]

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@
       --celestia.gas-multiplier $GAS_MULTIPLIER \
       --celestia.namespace-id $NAMESPACEID \
       --celestia.rpc $CELESTIA_NODE_ENDPOINT \
-      --celestia.keyring-keyname $KEYNAME
+      --celestia.keyname $KEYNAME
 ```
 
 ## Flags
@@ -33,7 +33,7 @@ Usage of daserver:
       --celestia.enable                                    Enable Celestia DA
       --celestia.gas-multiplier float                      Gas multiplier for Celestia transactions (default 1.01)
       --celestia.gas-price float                           Gas for retrying Celestia transactions (default 0.01)
-      --celestia.keyring-keyname string                    Celestia DA node keyring keyname for blobs submissions
+      --celestia.keyname string                            Celestia DA node keyring keyname for blobs submissions
       --celestia.namespace-id string                       Celestia Namespace to post data to
       --celestia.noop-writer                               Noop writer (disable posting to celestia)
       --celestia.read-auth-token string                    Auth token for Celestia Node

--- a/das/celestia.go
+++ b/das/celestia.go
@@ -10,6 +10,7 @@ import (
 	"math/big"
 	"strings"
 	"time"
+	"regexp"
 
 	"github.com/spf13/pflag"
 
@@ -38,7 +39,7 @@ type DAConfig struct {
 	NamespaceId        string           `koanf:"namespace-id" `
 	AuthToken          string           `koanf:"auth-token" reload:"hot"`
 	ReadAuthToken      string           `koanf:"read-auth-token" reload:"hot"`
-	KeyringKeyname     string     	    `koanf:"keyring-keyname" reload:"hot"`
+	KeyName     	   string     		`koanf:"keyname" reload:"hot"`
 	NoopWriter         bool             `koanf:"noop-writer" reload:"hot"`
 	ValidatorConfig    *ValidatorConfig `koanf:"validator-config"`
 	ReorgOnReadFailure bool             `koanf:"dangerous-reorg-on-read-failure"`
@@ -92,6 +93,7 @@ type CelestiaDA struct {
 	ReadClient *openrpc.Client
 	Namespace  *share.Namespace
 	Prover     *CelestiaProver
+	KeyName    string
 }
 
 type CelestiaProver struct {
@@ -109,7 +111,7 @@ func CelestiaDAConfigAddOptions(prefix string, f *pflag.FlagSet) {
 	f.String(prefix+".namespace-id", "", "Celestia Namespace to post data to")
 	f.String(prefix+".auth-token", "", "Auth token for Celestia Node")
 	f.String(prefix+".read-auth-token", "", "Auth token for Celestia Node")
-	f.String(prefix+".keyring-keyname", "", "Keyring keyname for Celestia Node for blobs submission")
+	f.String(prefix+".keyname", "", "Keyring keyname for Celestia Node for blobs submission")
 	f.Bool(prefix+".noop-writer", false, "Noop writer (disable posting to celestia)")
 	f.String(prefix+".validator-config"+".tendermint-rpc", "", "Tendermint RPC endpoint, only used for validation")
 	f.String(prefix+".validator-config"+".eth-rpc", "", "L1 Websocket connection, only used for validation")
@@ -149,15 +151,11 @@ func NewCelestiaDA(cfg *DAConfig, ethClient *ethclient.Client) (*CelestiaDA, err
 		return nil, err
 	}
 
-    if cfg.KeyringKeyname == "" {
+    if cfg.KeyName == "" {
         return nil, errors.New("keyring keyname cannot be blank")
     }
-	if !isValidKeyringKeyname(cfg.KeyringKeyname) {
-        return nil, fmt.Errorf("invalid keyring keyname format: %s", cfg.KeyringKeyname)
-    }
-	err = daClient.SetKeyringKeyname(cfg.KeyringKeyname)
-    if err != nil {
-        return nil, fmt.Errorf("failed to set keyring keyname: %w", err)
+    if !isValidKeyName(cfg.KeyName) {
+        return nil, fmt.Errorf("invalid keyring keyname format: %s", cfg.KeyName)
     }
 
 	if cfg.ValidatorConfig != nil {
@@ -191,6 +189,7 @@ func NewCelestiaDA(cfg *DAConfig, ethClient *ethclient.Client) (*CelestiaDA, err
 			Client:     daClient,
 			ReadClient: readClient,
 			Namespace:  &namespace,
+			KeyName:    cfg.KeyName,
 			Prover: &CelestiaProver{
 				Trpc:        trpc,
 				EthClient:   ethRpc,
@@ -718,7 +717,7 @@ func (c *CelestiaDA) returnErrorHelper(err error) (*ReadResult, error) {
 	return nil, err
 }
 
-// Validate that the KeyringKeyname is a alphanumeric string of length > 0
-func isValidKeyringKeyname(name string) bool {
+// Validate that the KeyName is a alphanumeric string of length > 0
+func isValidKeyName(name string) bool {
     return len(name) > 0 && regexp.MustCompile(`^[a-zA-Z0-9_-]+$`).MatchString(name)
 }

